core/app/models/spree/shipment.rb

```diff
@@ -249,8 +249,7 @@ def selected_shipping_rate_id
     end
 
     def selected_shipping_rate_id=(id)
-      shipping_rates.update_all(selected: false)
-      shipping_rates.touch_all # Bust cache dependent on "updated_at" timestamp
+      shipping_rates.update_all(selected: false, updated_at: Time.current)
```

Comment:
I'd leave a note that we're updating updated_at, to burst caches that cached incorrect `selected` value :)

Reply:
Yeah, let's leave the comment.